test/services/link/index.test.js

```diff
@@ -15,7 +23,53 @@ describe('link service', () => {
 		this.timeout(10000);
 
 		const url = 'https://schul-cloud.org/';
-		return service.create({ target: url })
+		return service.create({ target: url, forceNew })
+			.then((data) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=3811219)